js/src/components/ContentNavigation.js

```diff
@@ -102,25 +102,38 @@ const ContentNavigation = ({
                 <Button
                   {...buttonProps}
                   onClick={() => history.push({ pathname: `/section-${currentIndex + 2}` })}
+                  aria-label={Drupal.t('Next', {}, { context: 'ANU LMS' })}
```

Comment:
I'm not sure if it's the right way to use aria-label. It doesn't make sense from a11y point of view if the button already contains the same text.

Reply:
Remove the aria-labels in favor of data-test attributes